client/menu/menu.js

```diff
@@ -13,7 +13,17 @@
 // limitations under the License.
 
 /**
- * @fileoverview Module for storing question and user data
+ * @fileoverview Adds an importable configuration file for the TIE menu page.
  */
 
-window.tieData = angular.module('tieData', []);
+window.tieMenu = angular.module('tieMenu', ['tieData']);
+
+tieMenu.controller('MenuPageController',
+  ['$scope', 'QuestionDataService', function($scope, QuestionDataService) {
+    var questionSetId = 'strings';
+    QuestionDataService.initCurrentQuestionSet(questionSetId);
+    $scope.questionSet = QuestionDataService.getCurrentQuestionSet(
```

Comment:
You can do a console.log for the questionSet or the questionIds here to see it grabbing correct values. I had to remove because linter (and we wouldn't want console logs in our production code anyway).